tests/raftstore/server.rs

```diff
@@ -110,6 +118,41 @@ impl Simulator for ServerCluster {
         let raft_router = ServerRaftStoreRouter::new(store_sendch.clone(), snap_status_sender);
         let sim_router = SimulateTransport::new(raft_router);
 
+        // Create engine
```

Comment:
Is it the same as `tests/raftstore/node.rs`? Can we extract a function to create engines?

Reply:
I have the same question.